pkg/config/config.go

```diff
@@ -268,6 +268,16 @@ func getUriContent(uri string) ([]byte, error) {
 	return body, nil
 }
 
+func ImportRepositories(cfg *Config, l log.Logger) error {
```

Comment:
This function never returns an error.

Reply:
Yes, you are right, changed.
I had mixed feelings about breaking or not the process in case of an error.
In the end I think it's better to break it, as if you want to rely on the tool to check for your assets, if one of the configured repositories is not available and we skip the error, the process could end without detecting errors just because it couldn't load the checktypes.

Reply:
I removed this change from this pr and moved here https://github.com/adevinta/vulcan-local/commit/0a20c0ac012a0424e9e4d46f05c5bda31fd88ef5